tests/unit/test_data_quality.py

```diff
@@ -258,6 +259,7 @@ def test_upsert_ruleset(df: pd.DataFrame, glue_database: str, glue_table: str, g
     assert row_count.iloc[0]["expression"] == "between 2 and 8"
 
 
+@pytest.mark.xfail(raises=botocore.exceptions.ClientError, reason="Service error when evaluating multiple rulesets.")
```

Comment:
Unrelated to this change. Glue DQ multi-ruleset test case started failing on all branches. Asked service team for the  clarification, skipping in the meantime.